ld_script.txt

```diff
@@ -105,6 +105,7 @@ SECTIONS {
         src/random.o(.text);
         src/util.o(.text);
         src/daycare.o(.text);
+        src/debug.o(.text);
```

Comment:
Not necessary as per https://github.com/pret/pokeemerald/pull/1738

Reply:
removed